skills_DS/api/views.py

```diff
@@ -0,0 +1,28 @@
+from django.shortcuts import render
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.parsers import FileUploadParser
+
+# Create your views here.
+class AnswersView(APIView):
+	def post(self, request, format=None):
+		if request.data:
+			print(request.data['age'], request.data['gender'], request.data['yearOfStudy'])
+			return Response({'hey': 'it worked'}, status=status.HTTP_200_OK)
```

Comment:
If you write code that is intended to change later, leave TODO comments to make that intent obvious